recipes/cufflinks/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cufflinks" %}
+{% set version = "0.14.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b0413ef57b56390b8da6faf8e90172ac73772e0d4146ff80c968f22890f293a3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
```

Comment:
please remove this as you are not using it

Reply:
Done!